src/trigger.js

```diff
@@ -0,0 +1,189 @@
+          m(".progress-bar.progress-bar-striped.active[aria-valuemax='100'][aria-valuemin='0'][aria-valuenow='100'][role='progressbar']", {style: {"width": "100%"}}, [
+            m("span.sr-only", "Now Loading")
+          ])
+        ])
+      ]);
+    }
+
+    return state.gitlab.projects.filter((project) =>{
+      return matchSearchKey(project);
+    }).map((project) => {
+      project.events = project.events || projectEvents(project);
+
+      const names = [];
+      var avatar = "../img/gitlab_logo_48.png";
+      if(project.avatar_url) {
+        avatar = project.avatar_url;
+      }
+      names.push(m("img.icon.img-rounded", {src: avatar}));
+      names.push(
+        m('button.btn[id="projName"]', {
+          onclick: (e) => {
+            var targ = e.target;
+            $( "#curProjName" ).html( targ.innerText );
+            state.gitlab.loadBranchs( targ.innerText );
+            e.preventDefault();
+          }
+        }, 
+        project.path_with_namespace
+        ) );
+      return m("li.name", m("div",names));
+    });
+  };
+
+  return m("div", [
+    m("h2", [
+      m("i.fa.fa-gitlab[aria-hidden='true']"),
+      " GitLab CI trigger",
+    ]),    
+    m("span.status", state.status_message),
+    m("h2", "Repository List"),
+    m("form.form-inline[role='form']", [
+      m(".form-group.col-xs-4", [
+        m(".input-group", [
+          m(".input-group-addon", [
+            m("span.glyphicon.glyphicon-search")
+          ]),
+          m("input.form-control[id='search_repository'][placeholder='Project name'][type='text']", {
+            oninput: m.withAttr("value", (value) => { state.search_key = value; }),
+            value: state.search_key,
+          })
+        ])
+      ]),
+      m("button.btn.btn-info", {
+        onclick: (event) => {
+          state.reloadGitLabFromConfig();
+          state.gitlab.loadProjects();
+          event.preventDefault();
+        }
+      }, [
+        m("span.glyphicon.glyphicon-refresh"),
+        " Refresh Repository List"
+      ])
+    ]),
+    m("div.row", [
+      m("nav.col-sm-4[id=porjNav]", [
+        m("ul.nav.nav-pills.nav-stacked", projList()),
+      ]),
+      m("nav.col-sm-6[id=porjNav]", [
+        m("div.row.form-group", [
+          m("nav.col-sm-3", m("button.btn.btn-primary.btn-block","project")),
+          m("nav.col-sm-7", m("button.btn.btn-block[id=curProjName]","Please select a Project")),
+        ]),
+        m("div.row.form-group", [
+          m("nav.col-sm-3", m("div",tokenBtn())),
+          m("nav.col-sm-7", m("button.btn.btn-block[id=curToken]","Please select a token")),
+        ]),
+        m("div.row.form-group", [
+          m("nav.col-sm-3", m("div",branchBtn())),
+          m("nav.col-sm-7", m("button.btn.btn-block[id=curBranch]","Please select a branch")),
+        ]),
+        m("div.row.form-group", [
+          m("button.btn-block.btn-primary.btn",{onclick: (e) => {
+            var data = { "token": $( "#curToken" ).html(),
```

Comment:
Same to https://github.com/sue445/chrome-gitlab-notifier/pull/124/files#r152980351